lib/models.ts

```diff
@@ -38,6 +38,7 @@ export interface IGenerateModelsConfig {
     isEnterpriseSubscription: boolean;
     sdkType: SdkType;
     apiKey: string;
+    managementApiUrl?: string;
```

Comment:
I intentionally left it, because I consider this an internal configuration key that most users outside Kontent.ai employees won't need. That being said there isn't much harm in adding it either.